internal/daemonset/daemonset.go

```diff
@@ -373,34 +373,46 @@ func MakeLoadCommand(spec kmmv1beta1.ModprobeSpec, modName string) []string {
 		"-c",
 	}
 
-	loadCommand := "modprobe"
+	var loadCommand strings.Builder
 
-	if ra := spec.RawArgs; ra != nil && len(ra.Load) > 0 {
-		loadCommand = fmt.Sprintf("%s %s", loadCommand, strings.Join(ra.Load, " "))
-		return append(loadCommandShell, loadCommand)
+	if rawArgs := spec.RawArgs; rawArgs != nil && len(rawArgs.Load) > 0 {
+		loadCommand.WriteString("modprobe")
```

Comment:
one question: why RawArgs cannot be used together with firmware upload?

Reply:
Fair point. Should we cover that in a follow-up PR?

Reply:
I think we can change it in this one, it is not a big change